src/plugins/ffe/ffe_access_database.py

```diff
@@ -20,7 +19,7 @@ def __init__(self, file: Path, write: bool = False):
 
     def read_player_dicts(self) -> Iterator[dict[str, str | int | datetime | None]]:
         # the fields common to players with a club or not
-        common_fields: dict[str, tuple[str, types.FunctionType]] = {
+        common_fields: dict[str, str | int | datetime | None] = {
```

Comment:
Looks like it should be 
```suggestion
        common_fields: dict[str, str] = {
```